charts/hush-sensor/templates/_helpers.tpl

```diff
@@ -103,6 +103,21 @@ type: Unconfined
     {{- end -}}
 {{- end }}
 
+{{/*
+Hush deployment info
+*/}}
+{{- define "hush-sensor.deploymentInfo" -}}
+{{- $parts := .Values.deploymentToken | b64dec | split ":" -}}
```

Comment:
Where did you find documentation for `split`?
I found `regexSplit` only.

Reply:
https://masterminds.github.io/sprig/strings.html

sprig seems to be a dependency of helm.

Reply:
No `split` in the link.

Reply:
Ack, found here https://masterminds.github.io/sprig/string_slice.html